Floofbot/Modules/Help.cs

```diff
@@ -73,5 +75,74 @@ public async Task HelpCommand()
                 Trash = true
             });
         }
+
+        [Summary("Show help for a specific module")]
+        [Command("help")]
+        public async Task HelpCommand([Summary("module")]string requestedModule)
+        {
+            try
```

Comment:
Not sure why we need this try catch here.

Reply:
left over from testing, my bad